reducers/currentFlyEntry.js

```diff
@@ -5,9 +5,14 @@ export const currentFlyEntry = (state = {
 	category: '',
 	amount: '' }, action) => {
 	switch(action.type) {
+		case 'SET_CURRENT_FLY':
+			state = action.data
+			return state
+
```

Comment:
This was a great idea to set the state to the whole Fly object.  I first thought to go line by line of the object property, but this way help make the code look cleaner.  Good catch on this setup!